### PR TITLE
Remove disabled property and use quarantine instead

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: src/views/Workers/WorkerManager.js [hunk 1]
```diff
@@ -235,8 +228,10 @@ export default class WorkerManager extends React.PureComponent {
                       <td>
                         <Label
                           bsSize="sm"
-                          bsStyle={worker.disabled ? 'danger' : 'success'}>
-                          {worker.disabled ? 'disabled' : 'enabled'}
+                          bsStyle={
+                            worker.quarantineUntil ? 'danger' : 'success'
+                          }>
+                          {worker.quarantineUntil ? 'yes' : 'no'}
```
Comment (helfi92) [inline src/views/Workers/WorkerManager.js]: I'm open to something different than `yes` and `no`.

**Comment by helfi92** [comment]
Once I get an OK on this pull-request, I will merge https://github.com/taskcluster/taskcluster-queue/pull/222 before merging this one.

File: src/views/Workers/WorkerManager.js (hunk 1 above)
Comment (djmitche) [reply src/views/Workers/WorkerManager.js]: I think this is fine, but maybe a moment.js relative time?

"Quarantined: 13 days"

**Comment by helfi92** [comment]
I will merge after the release of Firefox Quantum to mitigate the risks.

File: src/views/Workers/WorkerManager.js (hunk 1 above)
Comment (helfi92) [reply src/views/Workers/WorkerManager.js]: Great idea! I will add it in a subsequent PR so we can mitigate any weird errors. Right now It's a little bit annoying to test without having https://github.com/taskcluster/taskcluster-queue/pull/222 merged first.

**Pull request merged**